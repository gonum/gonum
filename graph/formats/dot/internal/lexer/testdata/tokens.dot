# C preprocessing directives act as comments.
/* block comment */
// keywords are case-insensitive.
graph {
	node []
	Node []
	NODE []
	edge []
	Edge []
	EDGE []
	subgraph {}
	subGraph {}
	Subgraph {}
	SubGraph {}
	SUBGRAPH S {}
	A; B [style=filled, fillcolor=red]
	C:nw -- D:se
	"foo"
	.10
	-20
	3.14
	F [label=<<div>foo</div>>]
	H [label=<<div>â†’</div>>]
	_foo
	a10
}
Graph {
}
GRAPH {
}
digraph {
}
Digraph {
}
diGraph {
}
DiGraph {
}
DIGRAPH {
}
