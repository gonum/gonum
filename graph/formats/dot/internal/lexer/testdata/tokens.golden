graph {
	node []
	node []
	node []
	edge []
	edge []
	edge []
	{}
	{}
	{}
	{}
	subgraph S {}
	A
	B [style=filled fillcolor=red]
	C:nw -- D:se
	"foo"
	.10
	-20
	3.14
	F [label=<<div>foo</div>>]
	H [label=<<div>â†’</div>>]
	_foo
	a10
}
graph {
}
graph {
}
digraph {
}
digraph {
}
digraph {
}
digraph {
}
digraph {
}
